xml/System.Text/Encoding.xml

```diff
@@ -65,150 +65,153 @@
-|20000|x-Chinese-CNS|Chinese Traditional (CNS)||  
-|20001|x-cp20001|TCA Taiwan||  
-|20002|x-Chinese-Eten|Chinese Traditional (Eten)||  
-|20003|x-cp20003|IBM5550 Taiwan||  
-|20004|x-cp20004|TeleText Taiwan||  
-|20005|x-cp20005|Wang Taiwan||  
-|20105|x-IA5|Western European (IA5)||  
-|20106|x-IA5-German|German (IA5)||  
-|20107|x-IA5-Swedish|Swedish (IA5)||  
-|20108|x-IA5-Norwegian|Norwegian (IA5)||  
-|20127|us-ascii|US-ASCII|✓|  
-|20261|x-cp20261|T.61||  
-|20269|x-cp20269|ISO-6937||  
-|20273|IBM273|IBM EBCDIC (Germany)||  
-|20277|IBM277|IBM EBCDIC (Denmark-Norway)||  
-|20278|IBM278|IBM EBCDIC (Finland-Sweden)||  
-|20280|IBM280|IBM EBCDIC (Italy)||  
-|20284|IBM284|IBM EBCDIC (Spain)||  
-|20285|IBM285|IBM EBCDIC (UK)||  
-|20290|IBM290|IBM EBCDIC (Japanese katakana)||  
-|20297|IBM297|IBM EBCDIC (France)||  
-|20420|IBM420|IBM EBCDIC (Arabic)||  
-|20423|IBM423|IBM EBCDIC (Greek)||  
-|20424|IBM424|IBM EBCDIC (Hebrew)||  
-|20833|x-EBCDIC-KoreanExtended|IBM EBCDIC (Korean Extended)||  
-|20838|IBM-Thai|IBM EBCDIC (Thai)||  
-|20866|koi8-r|Cyrillic (KOI8-R)||  
-|20871|IBM871|IBM EBCDIC (Icelandic)||  
-|20880|IBM880|IBM EBCDIC (Cyrillic Russian)||  
-|20905|IBM905|IBM EBCDIC (Turkish)||  
-|20924|IBM00924|IBM Latin-1||  
-|20932|EUC-JP|Japanese (JIS 0208-1990 and 0212-1990)||  
-|20936|x-cp20936|Chinese Simplified (GB2312-80)|✓|  
-|20949|x-cp20949|Korean Wansung|✓|  
-|21025|cp1025|IBM EBCDIC (Cyrillic Serbian-Bulgarian)||  
-|21866|koi8-u|Cyrillic (KOI8-U)||  
-|28591|iso-8859-1|Western European (ISO)|✓|  
-|28592|iso-8859-2|Central European (ISO)||  
-|28593|iso-8859-3|Latin 3 (ISO)||  
-|28594|iso-8859-4|Baltic (ISO)||  
-|28595|iso-8859-5|Cyrillic (ISO)||  
-|28596|iso-8859-6|Arabic (ISO)||  
-|28597|iso-8859-7|Greek (ISO)||  
-|28598|iso-8859-8|Hebrew (ISO-Visual)|✓|  
-|28599|iso-8859-9|Turkish (ISO)||  
-|28603|iso-8859-13|Estonian (ISO)||  
-|28605|iso-8859-15|Latin 9 (ISO)||  
-|29001|x-Europa|Europa||  
-|38598|iso-8859-8-i|Hebrew (ISO-Logical)|✓|  
-|50220|iso-2022-jp|Japanese (JIS)|✓|  
-|50221|csISO2022JP|Japanese (JIS-Allow 1 byte Kana)|✓|  
-|50222|iso-2022-jp|Japanese (JIS-Allow 1 byte Kana - SO/SI)|✓|  
-|50225|iso-2022-kr|Korean (ISO)|✓|  
-|50227|x-cp50227|Chinese Simplified (ISO-2022)|✓|  
-|51932|euc-jp|Japanese (EUC)|✓|  
-|51936|EUC-CN|Chinese Simplified (EUC)|✓|  
-|51949|euc-kr|Korean (EUC)|✓|  
-|52936|hz-gb-2312|Chinese Simplified (HZ)|✓|  
-|54936|GB18030|Chinese Simplified (GB18030)|✓|  
-|57002|x-iscii-de|ISCII Devanagari|✓|  
-|57003|x-iscii-be|ISCII Bengali|✓|  
-|57004|x-iscii-ta|ISCII Tamil|✓|  
-|57005|x-iscii-te|ISCII Telugu|✓|  
-|57006|x-iscii-as|ISCII Assamese|✓|  
-|57007|x-iscii-or|ISCII Oriya|✓|  
-|57008|x-iscii-ka|ISCII Kannada|✓|  
-|57009|x-iscii-ma|ISCII Malayalam|✓|  
-|57010|x-iscii-gu|ISCII Gujarati|✓|  
-|57011|x-iscii-pa|ISCII Punjabi|✓|  
-|65000|utf-7|Unicode (UTF-7)|✓|  
-|65001|utf-8|Unicode (UTF-8)|✓|  
+|37|IBM037|IBM EBCDIC (US-Canada)|||  
```

Comment:
>37|IBM037|IB [](start = 1, length = 12)

I assume you got this list by running some code enumerating the encodings. right?

Reply:
Yes, I did, @tarekgh.